dowhy/gcm/util/plotting.py

```diff
@@ -0,0 +1,137 @@
+from typing import Optional, Dict, Tuple, Any
+
+import networkx as nx
+import pandas as pd
+from matplotlib import pyplot
+from networkx.drawing import nx_pydot
+
+
+def plot(causal_graph: nx.Graph,
+         causal_strengths: Optional[Dict[Tuple[Any, Any], float]] = None,
+         filename: Optional[str] = None,
+         display_plot: bool = True,
+         **kwargs) -> None:
+    """Convenience function to plot causal graphs. This function uses different backends based on what's
+    available on the system. The best result is achieved when using Graphviz as the backend. This requires both
+    the Python graphviz package (``pip install graphviz``) and the shared system library (e.g. ``brew install
+    graphviz`` or ``apt-get install graphviz``). When graphviz is not available, it will fall back to the
+    networkx backend.
+
+    :param causal_graph: The graph to be plotted
+    :param causal_strengths: An optional dictionary with Edge -> float entries.
+    :param filename: An optional filename if the output should be plotted into a file.
+    :param display_plot: Optionally specify if the plot should be displayed or not (default to True).
+    :param kwargs: Remaining parameters will be passed through to the backend verbatim.
+
+    **Example usage**::
+
+    >>> plot(nx.DiGraph([('X', 'Y')])) # plots X -> Y
+    >>> plot(nx.DiGraph([('X', 'Y')]), causal_strengths={('X', 'Y'): 0.43}) # annotates arrow with 0.43
+    """
+    try:
+        from causality.util.graphviz import plot_causal_graph_graphviz
+        from graphviz import ExecutableNotFound
+        try:
+            plot_causal_graph_graphviz(causal_graph,
+                                       causal_strengths=causal_strengths,
+                                       filename=filename,
+                                       display_plot=display_plot,
+                                       **kwargs)
+        except ExecutableNotFound:
+            _plot_causal_graph_networkx(causal_graph,
```

Comment:
odd to have _ for networkx plot but not for graphviz

Reply:
You are right, changed the graphviz version to "_" as well. I think the main reason was that the graphviz plot function is in a different optional module. But we can still simply import the "_" version.